src/NinjaTools.FluentMockServer.API/Proxy/ProxyMiddleware.cs

```diff
@@ -0,0 +1,58 @@
+using System.Threading.Tasks;
+using JetBrains.Annotations;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Options;
+using NinjaTools.FluentMockServer.API.Administration;
+using NinjaTools.FluentMockServer.API.Services;
+
+namespace NinjaTools.FluentMockServer.API.Proxy
+{
+    public class ProxyMiddleware
+    {
+        private readonly IOptions<AdminOptions> _adminOptions;
+        private readonly RequestDelegate _next;
+        private readonly ISetupService _setupService;
+        private readonly ILogService _logService;
+
+        public ProxyMiddleware(RequestDelegate next, ISetupService setupService, IOptions<AdminOptions> adminOptions, ILogService logService)
+        {
+            _next = next;
+            _setupService = setupService;
+            _adminOptions = adminOptions;
+            _logService = logService;
+        }
+
+        public async Task InvokeAsync([NotNull] HttpContext context)
+        {
+            var path = context.Request.Path;
+            var method = context.Request.Method;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'method'.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4802642)